test/e2e/egress_test.go

```diff
@@ -18,9 +18,14 @@ import (
 	"context"
 	"fmt"
 	"net"
+	"strings"
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
But I think you should still include the change to the patch permission of antrea-controller. Otherwise people who tests it will not get expected result.

Reply:
sure, done